.pre-commit-config.yaml

```diff
@@ -20,6 +20,7 @@ repos:
   hooks:
   - id: ruff
     types_or: [python, pyi, jupyter]
+    args: [--fix]
```

Comment:
Interesting, I have been in a split on this choice in the past as well. It is semi risky, I have had cases where the fix did break things. I'll leave it up to you which one you want

Reply:
I've removed it